### PR TITLE
Tidyup

### PR DESCRIPTION
**Pull request by marqh**
this puts the cf_units codebase in a state where it is a drop in replacement for iris.units

this will enable further development to happen alongside the management of the dependency relationship, if that is the direction we choose to take

### CODE REVIEW

**Comment by ocefpaf** [comment]
:+1:

Thanks @marqh!

**Comment by ocefpaf** [comment]
@marqh I am 100% OK with this change, but I do not feel comfortable merging this one.

@dkillick can you take a look and merge?

**Comment by DPeterK** [comment]
> @dkillick can you take a look and merge?

If you give me merge rights :wink:

**Comment by marqh** [comment]
So, the important facet for me for this to be merged is that the Iris branch in:
https://github.com/SciTools/iris/pull/1820

passes all of its tests when run with this

I think it does

**Comment by ocefpaf** [comment]
> If you give me merge rights

Sorry @dkillick. I though you already had.  Can you check if it work?  GitHub changed the way to manage repo rights.

> So, the important facet for me for this to be merged is that the Iris branch in

Let's get that working first then.  I will worry about a new version of #33 once the dust settle.

Thanks a lot @marqh I am happy to see `cf_units` folded back into iris!

**Comment by DPeterK** [comment]
Thanks @ocefpaf! 

> I do not feel comfortable merging this one

What is it about merging this in particular that you're not sure about?

**Comment by DPeterK** [comment]
PS. Yes I can indeed merge now :smile:

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# Installer logs
+pip-log.txt
+
+#shared objects
+*.so
+
+#config
+cf_units/etc/site.cfg
+
+# Created by editiors
```
Comment (DPeterK) [inline .gitignore]: I wish we could find the origin of "editiors" and correct it so it doesn't keep getting propagated :flushed:

**Comment by ocefpaf** [comment]
> What is it about merging this in particular that you're not sure about?

Familiarity with iris.  The goal is evaluate this in the light of https://github.com/SciTools/iris/pull/1820.

**Comment by DPeterK** [comment]
I'm provisionally :+1: with these changes but we do have some failures on the related Iris #1820 branch that need looking into. I'll do this tomorrow...

**Comment by DPeterK** [comment]
OK, I think we're good! Thanks @marqh @ocefpaf

**Comment by DPeterK** [comment]
@ocefpaf this seems to be a good point to make a new release of `cf_units` - what do you think? v0.2? v0.5?

**Comment by ocefpaf** [comment]
Maybe I should address https://github.com/SciTools/cf_units/pull/33#issuecomment-152206556 first...

**Comment by marqh** [comment]
> Maybe I should address #33 (comment) first...

@ocefpaf will #33 just rebase, with perhaps a conflict resolution for the iris:1813 update, which is already in this branch, and hence master?

**Comment by ocefpaf** [comment]
> @ocefpaf will #33 just rebase, with perhaps a conflict resolution for the iris:1813 update, which is already in this branch, and hence master?

Probably.  I will take a look at that tomorrow.  (I am gone for the rest of the day today.)

**Comment by DPeterK** [comment]
> Maybe I should address #33 (comment) first...

:+1:

**Pull request merged**